js-syntax-series/function-declarations.md

```diff
@@ -228,7 +228,7 @@ console.log(fibCopy(3));
 // (Works as expected)
 ```
 
-This is as the fibonacci identifier for the function expression in the latter example shadows the variable from outside, and refers to the function, not the variable specified in the outer scope. As such it's never overwritten.
+As the last example shows the returned recursive call inside the fibonacci expression points to the actual named function expression, not the variable it is assigned to. That means when we call fibonacci recursively it will always point correctly, even though the variable name is reassigned.
```

Comment:
Good. I think I'll need to rethink some in this section and maybe explain it in more detail. I use terms like shadowing, which might also be unknown for many.